.github/workflows/minikube.yaml

```diff
@@ -0,0 +1,68 @@
+name: Minikube
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.x
+
+      - name: Set environment
+        run: |
+          echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
+          mkdir -p "$(go env GOPATH)/src/github.com/minio/"
+          ln -s "$PWD" "$(go env GOPATH)/src/github.com/minio/direct-csi"
+          echo "VERSION=$(git describe --tags --always --dirty)" >> $GITHUB_ENV
+
+      - name: Build binaries
+        env:
+          CGO_ENABLED: 0
+          GO111MODULE: on
+        run: |
+          go build -tags "osusergo netgo static_build" -ldflags="-X main.Version=${VERSION} -extldflags=-static" github.com/minio/direct-csi/cmd/direct-csi
+          go build -tags "osusergo netgo static_build" -ldflags="-X main.Version=${VERSION} -extldflags=-static" github.com/minio/direct-csi/cmd/kubectl-direct_csi
+
+      - name: Set build tag
+        run: |
+          echo "BUILD_TAG=v0.0.0-$(./direct-csi --version)" >> $GITHUB_ENV
+
+      - name: Push docker image
+        run: |
+          docker build -t "directcsighwf/direct-csi:${BUILD_TAG}" .
+          docker login --username directcsighwf --password 2e33b2b5-b6b9-458e-b05d-9be5e76dcd18
+          docker push "directcsighwf/direct-csi:${BUILD_TAG}"
+
```

Comment:
There is no need to push docker images to public docker.io you can simply push it to `minikube` docker-env

```
eval $(minikube docker-env)
```

```
docker build -t
```

This will make it available in the minikube environment the local docker image. https://minikube.sigs.k8s.io/docs/handbook/pushing/

Reply:
How does it work for direct-csi install? What is the org name and registry for direct-csi in this way?

Reply:
I am assuming if you already have the container built in locally i.e inside the minikube environment you don't need to publish it remotely.

And this container should be available on all simulated k8s nodes.

Reply:
Just refer direct-csi installation doc. This is not going to work.

Reply:
Did you even test what I am suggesting ?